README.md

```diff
@@ -50,7 +50,22 @@ Installation of WND-CHARM minimally requires a C++ compiler, LibTIFF and FFTW.
     * Some X11 libraries must be installed prior to compiling/installing PHYLIP (`make install` in the src dir.)
         * CentOS/RedHat: `sudo yum install libX11-devel libXt-devel libXaw-devel`
         * Ubuntu/Debian: `sudo apt-get install libX11-dev libxt-dev libxaw7-dev`
-
+* [OME-Files](https://gitlab.com/codelibre/ome): An alternative to libtiff for reading the (tiled) tiff files 
```

Comment:
please remove this since OME-files is gone.

Reply:
Done